selfdrive/ui/ui.cc

```diff
@@ -255,6 +258,9 @@ void UIState::update() {
     watchdog_kick();
   }
   emit uiUpdate(*this);
+  double remaining = next_frame_time - millis_since_boot();
```

Comment:
Think we need to either take the remaining time into account, or just use a static QTimer at 50 ms intervals, either accounts for a long poll time + other processing like drawing